File: woodroid/src/main/java/me/gilo/woodroid/models/SettingOption.java
package me.gilo.woodroid.models;

import com.fasterxml.jackson.annotation.JsonProperty;

public class SettingOption {

    String id;
    String option;
    String description;
    String value;

    @JsonProperty("default")
    String default_value;
    String tip;
    String placeholder;
    String type;
    String options;
    String group_id;


}

File: woodroid/src/main/java/me/gilo/woodroid/models/SettingOption.kt
package me.gilo.woodroid.models

import com.fasterxml.jackson.annotation.JsonProperty

class SettingOption {

    var id: String
    var label: String
    var description: String
    var value: String

    @JsonProperty("default")
    var default_value: String
    var tip: String
    var placeholder: String
    var type: String
    var options: Map<String, String>
    var group_id: String
}
